src/wrappers/set_protocol/index.ts

```diff
@@ -17,7 +17,6 @@
 'use strict';
 
 import { AuthorizableWrapper } from './AuthorizableWrapper';
-import { BTCETHRebalancingManagerWrapper } from './BTCETHRebalancingManagerWrapper';
 import { ContractWrapper } from './ContractWrapper';
```

Comment:
For more efficient code, if we're exporting everything we're importing, we can do:

`export { ContractWrapper } from './ContractWrapper';`